diff --git a/Makefile b/Makefile
index e275ee40..eacb6257 100644
--- a/Makefile
+++ b/Makefile
@@ -67,10 +67,11 @@ install: all
 	@if [ x"$(PREFIX)" = x ]; then echo "please define PREFIX"; exit 1; fi
 	mkdir -p $(PREFIX)/bin
 	cp _build/default/src/Karamel.exe $(PREFIX)/bin/krml
+	cp _build/default/src/Karamel.exe $(PREFIX)/krml
 	mkdir -p $(PREFIX)/include
 	cp -r include/* $(PREFIX)/include
-	mkdir -p $(PREFIX)/lib/krml
-	cp -r krmllib/* $(PREFIX)/lib/krml
+	mkdir -p $(PREFIX)/krmllib
+	cp -r krmllib/* $(PREFIX)/krmllib
 	mkdir -p $(PREFIX)/lib/krml/runtime
 	cp -r runtime/* $(PREFIX)/lib/krml/runtime
 	mkdir -p $(PREFIX)/share/krml/examples
